### DISCUSSION TITLE
Error: Unable to unpack repo: not enough space on disk

### DISCUSSION THREAD

**Discussion by ojasuno** [vercel/vercel#8334] [Help]
context:
I am using [Nextra](https://nextra.vercel.app/) here for his static website.

The repo and deployment was working okay, last night.

Today, began to upload some more code and seeing this error, during CI/CD deployment by Vercel.

error:
```
Error: Unable to unpack repo: not enough space on disk
```

Is there any theoretical limits on how big or small a github repo  should be, for vercel integration, building, and deployment.

**Comment by amyegan** [comment]
HI @ojasuno. I believe all of the Nextra issues you're seeing are related. Let's continue the conversation in your other post https://github.com/vercel/vercel/discussions/8333
